.github/workflows/failing-weekly.yaml

```diff
@@ -0,0 +1,60 @@
+name: failing-weekly
+
+on:
+  workflow_dispatch:
+  schedule:
+    # at 3am weekly on Sunday
+    - cron: '0 3 * * 0'
+
+env:
+  PRISMA_TELEMETRY_INFORMATION: 'ecosystem-tests failing-weekly.yaml'
+  CI: 1
+
+defaults:
+  run:
+    # this makes windows use bash as well, which makes `...  >> $GITHUB_ENV` work there
+    shell: bash
+
+jobs:
+  databases:
+    strategy:
+      fail-fast: false
+      matrix:
+        database:
+          # It fails, see https://github.com/prisma/prisma/issues/15083
+          - mongodb-azure-cosmosdb
+        # Since it fails, we can test only library
```

Comment:
Why?

Reply:
Ah, not in "we are only able to test library", but "it is enough to test one of both"?

Reply:
It fails the same, the issue is not about the engine here.